edgelet/edgelet-http-workload/tests/dns-san.rs

```diff
@@ -0,0 +1,317 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Very nice test! We were missing this. We did have some integration tests at the HSM layer to ensure that certificates would be validated like a server would but this is a way nicer validation.

Reply:
Thanks! I'd like to add a negative test case as well as soon as I figure out how to run multiple tests with the HSM library.